engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
@geoffjentry  I'm not sure if I understand what you're saying then. Let's have a short Hangouts about that whenever you're free?